.github/workflows/checks.yaml

```
@@ -100,6 +77,15 @@ jobs:
         run: rapids-check-pr-job-dependencies "${IGNORED_JOBS}"
         env:
           IGNORED_JOBS: ${{ inputs.ignored_pr_jobs }}
+      - name: Check changed files
+        id: changed-files
+        if: ${{ inputs.enable_check_version_files_changed }}
+        uses: rapidsai/shared-actions/changed-files@changed-files
```

Make this change before merging:

```suggestion
        uses: rapidsai/shared-actions/changed-files@main
```